test/e2e/common/security_context.go

```diff
@@ -130,21 +126,22 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		}
 
 		/*
-		  Release : v1.12
-		  Testname: Security Context: readOnlyRootFilesystem=true.
-		  Description: when a container has configured readOnlyRootFilesystem to true, write operations are not allowed.
-		  This test is marked LinuxOnly since Windows does not support creating containers with read-only access.
+			Release : v1.14
+			Testname: Security Context, readOnlyRootFilesystem=true.
+			Description: Container is configured to run with readOnlyRootFilesystem to true which will force containers to run with a read only root file system.
+			Write Operation MUST NOT be allowed and Pod MUST be in Failed state.
 		*/
-		It("should run the container with readonly rootfs when readOnlyRootFilesystem=true [LinuxOnly] [NodeConformance]", func() {
+		framework.ConformanceIt("should run the container with readonly rootfs when readOnlyRootFilesystem=true [NodeConformance]", func() {
```

Comment:
Please do not remove the ``[LinuxOnly]`` label, test contains a SecurityContext, which is not working on Windows.

Reply:
Reverted to keep LinuxOnly tag.